code/Company.php

```diff
@@ -2,19 +2,41 @@
 
 namespace SilverStripe\FrameworkTest\Model;
 
+use SilverStripe\Assets\Image;
 use SilverStripe\Forms\DropdownField;
+use SilverStripe\ORM\HasManyList;
+use SilverStripe\ORM\ManyManyList;
 use SilverStripe\ORM\ValidationResult;
 use SilverStripe\ORM\DataObject;
 use SilverStripe\ORM\DB;
 use SilverStripe\AssetAdmin\Forms\UploadField;
 use SilverStripe\Versioned\Versioned;
+use RelationFieldsTestPage;
+use GridFieldTestPage;
 
+/**
+ *
+ * @property string $Name
+ * @property string $Category
+ * @property float $Revenue
+ * @property string $CEO
+ * @property int $RelationFieldsTestPageID
+ * @property int $GridFieldTestPageID
+ * @method RelationFieldsTestPage RelationFieldsTestPage()
+ * @method GridFieldTestPage GridFieldTestPageHasOne()
+ * @method HasManyList|Employee[] Employees()
+ * @method HasManyList|Image[] GroupPhotos()
+ * @method ManyManyList|Employee[] PastEmployees()
+ *
+ * @mixin Versioned
+ * @mixin RecursivePublishable
+ */
 class Company extends DataObject
 {
     private static $table_name = 'Company';
 
     private static $extensions = [
-        Versioned::class
+//        Versioned::class
```

Comment:
Is there a reason this is commented out and not just removed?

Reply:
I was testing something to make sure it worked with and without versioning. I'll undo that bit.